web/satellite/jest.config.js

```diff
@@ -1,6 +1,8 @@
 // Copyright (C) 2022 Storj Labs, Inc.
 // See LICENSE for copying information.
 
+process.env.TZ = 'UTC';
```

Comment:
This seems like an unrelated change to the pull request. Similarly, changing TZ in jest config doesn't seem like a clean solution.

Reply:
Hi Egon, this was added due to an issue when updating snapshots. Moby has added this in another change so I will remove it from ours on the next commit.